src/renderer/screens/Assets/Assets.tsx

```diff
@@ -51,28 +51,25 @@ export const Assets = () => {
   }, [firstActiveAccount, activeWallet]);
 
   const updateAccounts = (accounts: Account[]) => {
-    const result = accounts.length ? accounts : [];
-
-    setActiveAccounts(result);
+    setActiveAccounts(accounts.length ? accounts : []);
   };
 
-  const availableConnectionsAmount = Object.values(connections).filter(
-    (c) => c.connection.connectionStatus === ConnectionStatus.CONNECTED,
-  ).length;
-
-  const sortedChains = useMemo(
-    () =>
-      sortChains(
-        Object.values(connections).filter((c) => {
-          const isDisabled = c.connection.connectionType === ConnectionType.DISABLED;
-          const hasMultisigAccount = activeAccounts.some(isMultisig);
-          const hasMultiPallet = !hasMultisigAccount || c.connection.hasMultisigPallet !== false;
-
-          return !isDisabled && hasMultiPallet;
-        }),
-      ),
-    [availableConnectionsAmount, activeAccounts],
-  );
+  useEffect(() => {
```

Comment:
Switched to `useEffect` because we'll end up storing lots of sorted arrays in memory